pyrpod/MissionPlanner.py

```diff
@@ -609,4 +635,7 @@ def plot_thrust_envelope(self):
 
             fig.savefig("test" + str(firing[0]) + ".png")
             plt.show()
-        return
\ No newline at end of file
+        return
+
+    def calc_dv(self, v_e, m_o, m_f):

```

Comment:
I have a calc_delta_v with different parameters that is called when calculating propellant expenditure in the JFH